pkg/spire/spire.go

```diff
@@ -254,3 +284,206 @@ func (sc *SpireServerApiClient) Close() {
 		// Log error
 	}
 }
+
+func (sc *SpireServerApiClient) fetchSVID() (*x509svid.SVID, error) {
+	if sc.SVID != nil {
+		return sc.SVID, nil
+	}
+	xsvid, err := sc.workloadAPI.FetchX509SVID(sc.ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to fetch controller SVID: %s", err)
+	}
+	sc.SVID = xsvid
+	return sc.SVID, nil
+}
+
+func (sc *SpireServerApiClient) CheckValidated(rs []v1beta1.TaskRunResult, tr *v1beta1.TaskRun) error {
```

Comment:
nit: make func name specific to validating task run result, in case it gets mistaken for validation of taskrun

Reply:
fixed name.